core/main/src/service/apps/delegated_launcher_handler.rs

```diff
@@ -603,7 +604,12 @@ impl DelegatedLauncherHandler {
                     false
                 }
             })
-            .filter(|perm| ps.cap_state.generic.check_all(&vec![perm.clone()]).is_ok())
```

Comment:
why only supported is checked here?

Reply:
This is to get the user's consent before launching the app. It is not going to use the cap immediately. So it is enough to check if the cap is supported. Only at the time of invocation, we have check if it is available.